Content.Server/Atmos/Piping/Unary/EntitySystems/GasCanisterSystem.cs

```diff
@@ -153,7 +155,7 @@ private void OnCanisterChangeReleaseValve(EntityUid uid, GasCanisterComponent ca
             if (!CheckInteract(args.Session))
                 return;
 
-            _adminLogSystem.Add(LogType.CanisterValve, LogImpact.High, $"{(args.Session as IPlayerSession):player} set the valve on {uid} to {args.Valve:valveState}");
+            _adminLogSystem.Add(LogType.CanisterValve, LogImpact.High, $"{args.Session.AttachedEntity:player} set the valve on {uid} to {args.Valve:valveState}");
```

Comment:
Wouldn't the impact depend on whether there is a tank inside the canister or not?

Reply:
Good point, sec